wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewController.swift

```diff
@@ -47,26 +47,17 @@ extension ZMConversationType {
 }
 
 final class ProfileViewController: UIViewController {
-    let viewModel: ProfileViewControllerViewModel
     weak var viewControllerDismisser: ViewControllerDismisser?
+    var delegate: ProfileViewControllerDelegate?
```

Comment:
Usually delegates are `weak` and not retained, can we apply it here?

Reply:
Good catch